interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -59,6 +59,9 @@
 // default values for $this_message    
     $this_message = array('message'=>'','message_priority'=>3,'dueDate'=>'');
     $forwarding = false;
+
+// default values for Max words to input in a reminder
```

Comment:
Ok, so lets go with the current approach 
I do not think we should change the Db to 255 char since I fear that the message would not display well in the UI 
also removing one of "onKeyUp" or" onKeyDown" does problems

thanks